critiquebrainz/ws/review/views.py

```diff
@@ -196,7 +339,7 @@ def review_post_handler(user):
 
     :reqheader Content-Type: *application/json*
 
-    :json uuid entity_id: UUID of the release group that is being reviewed
+    :json uuid entity_id: UUID of the entity that is being reviewed
```

Comment:
Wanted confirmation on this as well.
I believe it should be entity as the ID can be for any of the entities (release_group' or 'event')